gnlse/visualization.py

```diff
@@ -43,26 +262,206 @@ def plot_wavelength_vs_distance(solver, WL_range=[400, 1350], ax=None,
                       where=(np.abs(solver.AW)**2 > 0)))
     WL = 2 * np.pi * c / solver.W  # wavelength grid
     WL_asc = np.flip(WL, )  # ascending order for interpolation
-    iis = np.logical_and(WL_asc > WL_range[0],
-                         WL_asc < WL_range[1])  # indices of interest
+    iio = np.logical_and(WL_asc > WL_range[0],
```

Comment:
I think that the name of the variable was firstly introduced by phytonic statemant 'is in', 'io' can be related with input-output, but I don't mind to change that

Reply:
Name 'iis' is later used for indexing z_slice, so we prefer to keep 'iio'.